pages/_app.js

```diff
@@ -1,20 +1,23 @@
 import "../styles/reset.scss";
 import "../styles/globals.scss";
-import { FirebaseAppProvider } from 'reactfire';
+import { FirebaseAppProvider } from "reactfire";
+import { DefaultLayout } from "../layouts/DefaultLayout";
 
 const firebaseConfig = {
   apiKey: "AIzaSyCvapodXG0DHiW3SebX8MrOyDms5RcC1mM",
   authDomain: "ioc-poc-a3533.firebaseapp.com",
   projectId: "ioc-poc-a3533",
   storageBucket: "ioc-poc-a3533.appspot.com",
   messagingSenderId: "248607905999",
-  appId: "1:248607905999:web:f7c340f4346552c9bb7fba"
+  appId: "1:248607905999:web:f7c340f4346552c9bb7fba",
 };
 
 function MyApp({ Component, pageProps }) {
   return (
     <FirebaseAppProvider firebaseConfig={firebaseConfig}>
-      <Component {...pageProps} />
+      <DefaultLayout>
```

Comment:
Is there only one default layout ever? Adding it page to page means that we have control at a page level. What's the motivation here?

Reply:
Following up from the conversation we had during a meeting - I found this information on how we can apply multiple default layouts but at the app level.  We'd simply specify which layout as a property on the page.

https://simplernerd.com/nextjs-multiple-layouts/

What do you think about this approach for a future refactor?

Reply:
I had not seen that before, very interesting. I think it's a viable path forward if the default layout, that defines most of your app, for some reason doesn't work for one small use case and I think that's what our situation is. Before, I was hesitant to commit and give up control at the page level without knowing more about the website  plan. This solution outlined above is the "escape hatch" that we don't need to use but can use it to add novel layouts without needing to touch existing pages. 

I think the default layout should go at _app.js level too now.